sdk/go/common/workspace/plugins_test.go

```diff
@@ -1745,7 +1755,7 @@ func TestNewPluginSpec(t *testing.T) {
 				Name:              "github.com_pulumi_pulumi-example",
```

Comment:
To be able to distinguish the plugin.  If it's just `pulumi-example`, it could come from `gitlab.com/tgummerer/pulumi-example`, but have the same name as `github.com/pulumi/pulumi-example` when that's downloaded using the current mechanisms.  Which would get very confusing, especially as we're using this name on the file system to find the plugin again.

Reply:
Yeah, I think we'll have to fix those though, otherwise this is getting messy with disambiguating :thinking: